sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuExec.scala

```diff
@@ -278,6 +280,26 @@ trait GpuExec extends SparkPlan with Arm {
     SEMAPHORE_WAIT_TIME -> createNanoTimingMetric(DEBUG_LEVEL, DESCRIPTION_SEMAPHORE_WAIT_TIME)
   )
 
+  // FIXME Will it be better to call this in child execs where UDFs may exist, which requires
+  // listing all the execs that could have UDFs, instead of always iterating the expressions
```

Comment:
UDFs can appear just about anywhere `Expression` appears, so I don't think tracking down every specific exec is going to work well in practice.

Reply:
Got it. Thanks a lot.